docs/src/content/docs/pt-br/reference/plugins.md

```diff
@@ -0,0 +1,162 @@
+O hook `setup` pode ser utilizado para atualizar a configuração do Starlight ou adicionar integrações ao Astro.
+
+Este hook é chamado com as seguintes opções:
+
+#### `config`
+
+**tipo:** `StarlightUserConfig`
+
+Um cópia somente-leitura da [Configuração do Starlight](/pt-br/reference/configuration/) fornecida pelo usuário.
+Essa configuração pode ter sido atualizada por outros plugins configurados antes desse.
+
+#### `updateConfig`
+
+**tipo:** `(newConfig: StarlightUserConfig) => void`
+
+Uma função de callback para atualizar a [Configuração do Starlight](/pt-br/reference/configuration/) fornecida pelo usuário.
+Forceça as chaves de configuração no nível-raiz que você quer sobrescrever.
+Para atualizar valores de configuração aninhados, você precisa fornecer todo o objeto aninhado.
+
+Para extender uma configuração existênte sem sobrescrevê-la, distribua os valores existentes no novo valor.
+No seguinte exemplo, uma nova conta de mídia [`social`](/pt-br/reference/configuration/#social) é adicionada à configuração existente distribuindo o objeto `config.social` no novo objeto `social`:
+
+```ts {6-11}
+// plugin.ts
+export default {
+  name: 'add-twitter-plugin',
+  hooks: {
+    setup({ config, updateConfig }) {
+      updateConfig({
+        social: {
+          ...config.social,
+          twitter: 'https://twitter.com/astrodotbuild',
+        },
+      });
+    },
+  },
+};
+```
+
+#### `addIntegration`
+
+**tipo:** `(integration: AstroIntegration) => void`
+
+Uma função de callback para adicionar uma [Integração Astro](https://docs.astro.build/pt-br/reference/integrations-reference/) requerida pelo plugin.
+No seguinte exemplo, o plugin primeiro verifica se a [Integração de React do Astro](https://docs.astro.build/pt-br/guides/integrations-guide/react/) está configurada, caso não esteja, usa `addIntegration()` para adicioná-la:
+
+```ts {14} "addIntegration,"
+// plugin.ts
+import react from '@astrojs/react';
+
+export default {
+  name: 'plugin-using-react',
+  hooks: {
+    plugin({ addIntegration, astroConfig }) {
+      const isReactLoaded = astroConfig.integrations.find(
+        ({ name }) => name === '@astrojs/react'
+      );
+
+      // Só adiciona a integração com o React se ela não estiver carregada.
+      if (!isReactLoaded) {
+        addIntegration(react());
+      }
+    },
+  },
+};
+```
+
+#### `astroConfig`
+
+**tipo:** `AstroConfig`
+
+Uma cópia somente-leitura da [Configuração do Astro](https://docs.astro.build/pt-br/reference/configuration-reference/) fornecida pelo usuário.
+
+#### `command`
+
+**tipo:** `'dev' | 'build' | 'preview'`
+
+O comando usado para executar o Starlight:
+
+- `dev` - O projeto é executado com `astro dev`
+- `build` - O projeto é executado com `astro build`
+- `preview` - O projeto é executado com `astro preview`
+
+#### `isRestart`
+
+**tipo:** `boolean`
+
+`false` quando o servidor dev é iniciado, `true` quando uma atualização é disparada.
```

Comment:
```suggestion
`false` quando o servidor de desenvolvimento é iniciado, `true` quando uma atualização é disparada.
```
To match with `getting-started.mdx` 
https://github.com/withastro/starlight/blob/9625926fe603d35d7cf096eba32c74bd469661b3/docs/src/content/docs/pt-br/getting-started.mdx#L49